src/operators/infinitepepo.jl

```diff
@@ -3,26 +3,32 @@
 
 Represents an infinite projected entangled-pair operator (PEPO) on a 3D cubic lattice.
 """
-struct InfinitePEPO{T<:PEPOTensor} <: InfiniteSquareNetwork{T,3}
+struct InfinitePEPO{T<:PEPOTensor}
     A::Array{T,3}
 
```

Comment:
Can you also add an "unchecked" constructor `InfinitePEPO{T}(A::Array{T,3})` here?